src-vue/src/views/PlayView.vue

```diff
@@ -32,6 +39,11 @@ export default defineComponent({
             <div v-if="northstarVersion !== ''" class="fc_changelog__link" @click="showChangelogPage">
                 (see patch notes)
             </div>
+            <br />
+            <div>
```

Comment:
Unsure about positioning of these tbh, left-hand part of this view relates to local Northstar install (version + launch button), stats feel a bit out of place.

Reply:
Kinda agree but also didn't know where else to put it ^^"

Reply:
I really have no other idea for where to put it, so let's put it there.